src/style.css

```
@@ -0,0 +1,59 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  margin-top: 10px;
+  margin-right: 20px;
+}
+
+.nav {
+  height: 60px;
+}
+
+.nav_list {
+  display: flex;
+}
+
+.nav_item {
+  display: flex;
+  margin: 0 10px;
```

```suggestion
  margin-right: 20px;
```
Use margin-right for every item and use `:last-child` to reset the margin for the last item of your list. Then tests will pass